apps/trustlab/src/app/preview/route.ts

```diff
@@ -0,0 +1,72 @@
+import type { PayloadRequest } from "payload";
+import { getPayload } from "payload";
+
+import { draftMode } from "next/headers";
+import { redirect } from "next/navigation";
+
+import configPromise from "@payload-config";
```

Comment:
is payload config a promise? I thought it is a json object🤔

Reply:
@koechkevin Payload exports both the `config` and `configPromise`.